package/boot/uboot-envtools/files/ipq40xx

```diff
@@ -25,7 +25,7 @@ ubootenv_mtdinfo () {
 		ubootenv_size=0x40000
 	fi
 
-	sectors=$(( $ubootenv_size / $mtd_erase ))
+	sectors=$(printf 0x%x $(( $ubootenv_size / $mtd_erase )))
```

Comment:
$ in $(()) is pointless. Shellcheck warns on this.

Reply:
thinks, fix it